shared/src/main/scala/pl/metastack/metatime/Time.scala

```diff
@@ -24,6 +24,13 @@ trait Hour extends Period with Component {
       case other: Hour => other.h == h
       case _ => false
     }
+	
+  def <(that: Hour): Boolean = {
```

Comment:
You just have to implement `compare` (from `Ordered`). Then, it will already provide all operators automatically.